charts/argo-workflows/values.yaml

```diff
@@ -711,7 +711,7 @@ server:
 
   # SSO configuration when SSO is specified as a server auth mode.
   sso:
-    # -- Create SSO configuration. If you set `true` , please also set `.Values.server.authMode` as `sso`.
+    # -- Create SSO configuration. If you set `true` , please also set `.Values.server.authModes` as `sso`.
```

Comment:
I think you need to handle below to update README, by running `./scripts/helm-docs.sh` .
Ref:
> I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)

Reply:
Sorry I hit the command but the generation was falling because of this [problem](https://github.com/docker/for-mac/issues/7527). Fixed and generated now.